base/unicode/utf8.jl

```diff
@@ -101,26 +101,33 @@ sizeof(s::UTF8String) = sizeof(s.data)
 
 lastidx(s::UTF8String) = length(s.data)
 
+isfirstbyte(c::UInt8) = (c & 0xc0) != 0x80 # == !is_valid_continuation(c)
```

Comment:
`@code_llvm` says that `!is_valid_continuation(c)` compiles to code identical to `firstbyte(c)`, so I don't think we need a separate `firstbyte` function.